docs/doc_kr/README.md

```diff
@@ -0,0 +1,48 @@
+# PyScript
+
+## PyScript란?
+
+### Summary
+PyScript는 Scratch, JSFiddle 및 기타 "사용하기 쉬운" 프로그래밍 프레임워크에 대한 파이썬의 대안이며, 누구나 흥미롭고 상호작용하는 애플리케이션을 친근하고 나만의 방식대로 웹에 만드는 것을 목표로 합니다.
+
+시작하려면 [getting started tutorial](docs/tutorials/getting-started.md).
+
+예제보기 [the pyscript folder](pyscriptjs).
+
+### Longer Version
+PyScript는 사용자가 파이썬으로 정교한 브라우저 애플리케이션을 만들 수 있게 하는 여러 오픈소스 기술을 결합을 목표로 하는 메타 프로젝트입니다.
+PyScript는 브라우저에서 DOM이 작동하는 방식과 정교하게 결합하고 웹과 파이썬 개발자 모두에게 자연스럽게 파이썬 로직을 추가할 수 있습니다.
+
+
+## PyScript 사용하기
+
+PyScript을 사용하기 위해서, html 페이지에 올바른 pyscript 파일을 import 해야 합니다:
+```html
+<link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
+<script defer src="https://pyscript.net/alpha/pyscript.js"></script>
+```
+이제 PyScript 컴포넌트를 html page에 사용할 수 있습니다. PyScript는 현재 다음과 같은 요소(elements)들을 구현하고 있다.
```

Comment:
- Giving unity to the translation of repeated words.
- Giving unity to the use of honorifics.
```suggestion
이제 PyScript 컴포넌트들을 html 페이지에 사용할 수 있습니다. PyScript는 현재 다음과 같은 요소(elements)들을 구현하고 있습니다.
```